eng/Versions.props

```diff
@@ -47,37 +47,37 @@
   -->
   <PropertyGroup>
     <SystemCommandLineVersion>2.0.0-beta4.24528.1</SystemCommandLineVersion>
-    <SystemCompositionVersion>8.0.0</SystemCompositionVersion>
-    <SystemConfigurationConfigurationManagerVersion>8.0.0</SystemConfigurationConfigurationManagerVersion>
-    <SystemDiagnosticsEventLogVersion>8.0.0</SystemDiagnosticsEventLogVersion>
-    <SystemCollectionsImmutableVersion>8.0.0</SystemCollectionsImmutableVersion>
-    <SystemComponentModelCompositionVersion>8.0.0</SystemComponentModelCompositionVersion>
-    <SystemDiagnosticsDiagnosticSourceVersion>8.0.1</SystemDiagnosticsDiagnosticSourceVersion>
-    <SystemIOHashingVersion>8.0.0</SystemIOHashingVersion>
-    <SystemIOPipelinesVersion>8.0.0</SystemIOPipelinesVersion>
-    <SystemReflectionMetadataVersion>8.0.0</SystemReflectionMetadataVersion>
-    <SystemResourcesExtensionsVersion>8.0.0</SystemResourcesExtensionsVersion>
-    <SystemSecurityCryptographyProtectedDataVersion>8.0.0</SystemSecurityCryptographyProtectedDataVersion>
-    <SystemSecurityPermissionsVersion>8.0.0</SystemSecurityPermissionsVersion>
-    <SystemTextEncodingsWebVersion>8.0.0</SystemTextEncodingsWebVersion>
+    <SystemCompositionVersion>9.0.0</SystemCompositionVersion>
```

Comment:
Is it possible at all to deploy missing dependencies as part of F5/integration tests?  IIRC we have this project for it - https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/Setup.Dependencies/Roslyn.VisualStudio.Setup.Dependencies.csproj#L21